### PR TITLE
update: enable evaluation by set timer

### PR DESCRIPTION
**Pull request by zdtsw**
Ref: #473

### CODE REVIEW

File: pipelines/jobs/configurations/jdk11u.groovy [hunk 1]
```diff
@@ -1,7 +1,6 @@
 targetConfigurations = [
         'x64Mac'        : [    'temurin',    'openj9'                    ],
         'x64Linux'      : [    'temurin',    'openj9',    'dragonwell',    'corretto',    'bisheng',    'fast_startup'],
-        'x64AlpineLinux': [    'temurin'                            ],
```
Comment (gdams) [inline pipelines/jobs/configurations/jdk11u.groovy:4]: wrong platform? This PR says it moves the aarch64 alpine build?

Comment (zdtsw) [reply pipelines/jobs/configurations/jdk11u.groovy:4]: yes, my bad, will fix it

File: pipelines/jobs/configurations/jdk11u_evaluation.groovy [hunk 2]
```diff
@@ -1,4 +1,10 @@
 targetConfigurations = [
+        'aarch64Linux'  : [
```
Comment (gdams) [inline pipelines/jobs/configurations/jdk11u_evaluation.groovy:2]: still not Alpine Linux? perhaps I'm missing something here?

Comment (zdtsw) [reply pipelines/jobs/configurations/jdk11u_evaluation.groovy]: see it as my covid side effects.......

we actually do not have jdk11 testing  aarch64alpinelinux.
will not have it in evaluation.

revert back

Comment (zdtsw) [reply pipelines/jobs/configurations/jdk11u_evaluation.groovy]: there was some issue for jdk11 aarch64 alpine linux in the past, so it was disabled on jdk11
will first get other jdk version running and put it into evaluation later.

**Review by andrew-m-leonard — APPROVED** [review]

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**